### PR TITLE
update elsaaaa

### PR DESCRIPTION
**Pull request by diyapratheep123**

### CODE REVIEW

File: Vira ver 1.1.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
     else:
 
-        print("Access Denied")
+        print("Access denied")
```
Comment (georgerahul24) [inline Vira ver 1.1.py:63]: Make  this back into caps

File: Vira ver 1.1.py [hunk 2]
```diff
@@ -70,8 +70,8 @@
             time.sleep(2)
             exit()
         else:
-            talk("Access Denied")
-            talk("Please Try Again dear")
+            talk("Access denied")
+            talk("please try again")
```
Comment (georgerahul24) [inline Vira ver 1.1.py:74]: These also

File: Vira ver 1.1.py [hunk 3]
```diff
@@ -43,7 +43,7 @@
 
 print("Starting to verify the user")
 talk("Hello")
-talk("I am Vira version 1 point 1")
+talk("I am Elsa version 1 point 1")
```
Comment (georgerahul24) [inline Vira ver 1.1.py:46]: If possible remove the version 1.1 part

**Comment by georgerahul24** [comment]
@diyapratheep123 …
Please check these changes

**Comment by PARTHAN27** [comment]
Hey Gooys ..... Who is Elsa....🤣😂.... is that a short name for Eli?? Make it into Gokhuls Grexa😂 @georgerahul24

**Comment by georgerahul24** [comment]
> Hey Gooys ..... Who is Elsa....🤣😂.... is that a short name for Eli?? Make it into Gokhuls Grexa😂 @georgerahul24 

Elsa is for Elizabeth as well as a reference from 
```frozen.....```

**Comment by PARTHAN27** [comment]
> 
> 
> > Hey Gooys ..... Who is Elsa....🤣😂.... is that a short name for Eli?? Make it into Gokhuls Grexa😂 @georgerahul24
> 
> Elsa is for Elizabeth as well as a reference from
> `frozen.....`

Huh, is there a character named elsa in frozen .....best ....elizabeth d h github r8? @georgerahul24

**Comment by georgerahul24** [comment]
> > 
> 
> > 
> 
> > > Hey Gooys ..... Who is Elsa....🤣😂.... is that a short name for Eli?? Make it into Gokhuls Grexa😂 @georgerahul24
> 
> > 
> 
> > Elsa is for Elizabeth as well as a reference from
> 
> > `frozen.....`
> 
> 
> 
> Huh, is there a character named elsa in frozen .....best ....elizabeth d h github r8? @georgerahul24 

Yes and yes 😀

**Comment by georgerahul24** [comment]
Will this @diyapratheep123  come and make the changes?

**Comment by PARTHAN27** [comment]
> 
> 
> Will this @diyapratheep123 come and make the changes?
> Kure neerayalo...... ⏲️

Haa sookshicho .....ur vira ll b dead soooon😂😂😂

**Pull request closed without merging**